.github/workflows/linters.yml.txt

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
```

Comment:
- Hey, great adding the linter, however, you've added the `.txt` file extension to the `linters.yml`, you are not supposed to do that. please, remove it. After you've removed it, try to fix the linter error you'd encounter in the style.css file by formatting it correctly.

![image](https://github.com/umarahgul/Hello-world/assets/59397159/228d0066-aab9-4b7f-bf2c-9aee839fef9d)